### PR TITLE
docs(legal): add support service tos

### PR DESCRIPTION
**Pull request by mffap**
Adds Terms of Service for Support Services to the Legal section as well as the necessary annex, ie. updates the Support Service Annex, section for trainings to the Guides.

3rd level support is provided by CAOS for self-hosted instances.

### CODE REVIEW

File: docs/docs/guides/trainings/introduction.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Introduction
+---
+
+The following pages describe recommended training sessions for managing and supporting ZITADEL.
```
Comment (fforootd) [inline docs/docs/guides/trainings/introduction.md:5]:
```suggestion
The following pages describe the recommended training sessions for managing and supporting ZITADEL.
```

File: docs/docs/guides/trainings/supportservice/application.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Application Support Trainings
+---
+
+## ZITADEL Developer
```
Comment (fforootd) [inline docs/docs/guides/trainings/supportservice/application.md:5]: I find "developer" confusing... a training for developers might be geared towards the API and not the ZITADEL internals.

Maybe we should call this ZITADEL Internals?

File: docs/docs/guides/trainings/supportservice/operations.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+**Duration**: 1.5 days
+
+**Topics covered**:
+
+* Metrics
+* Logs
+* Monitoring
+* Alerting
+* Ingress/API-Gateway
+* GitOps
```
Comment (fforootd) [inline docs/docs/guides/trainings/supportservice/operations.md:62]: We provide a topic called gitops already with the basics, we should leave this here

File: docs/docs/guides/trainings/supportservice/recurring.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: Recurring Trainings
+---
+
+## ZITADEL Support Refresher
+
+In this session you can refresh knowledge about existing and gain experience with new features of ZITADEL to keep the quality of your support high. We recommend one day training per support staff.
```
Comment (fforootd) [inline docs/docs/guides/trainings/supportservice/recurring.md:7]:
```suggestion
In this session you can refresh knowledge about existing and gain experience with new features of ZITADEL to keep the quality of your support high. We recommend an one day training per support staff.
```

File: docs/docs/legal/terms-support-service.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
+---
+title: Terms and Conditions for Support Services
+custom_edit_url: null
+--- 
+## General
+
+### Introduction
+
+CAOS Ltd. (**"We"**, **CAOS AG**, or simply **CAOS**), with head office in Teufener Strasse 19, 9000 St. Gallen, Switzerland, offers amongst other services and websites (**Services**) as well commercial support services (**Support Services**) for units of CAOS software products.
```
Comment (fforootd) [inline docs/docs/legal/terms-support-service.md]:
```suggestion
CAOS Ltd. (**"We"**, **CAOS AG**, or simply **CAOS**), with head office located at Teufener Strasse 19, 9000 St. Gallen, Switzerland, offers amongst other services and websites (**Services**) as well commercial support services (**Support Services**) for units of CAOS software products.
```

File: docs/docs/legal/terms-support-service.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+
+Any provisions which deviate from these TOS must be agreed in writing between the Customer and us. Such agreements shall take precedence over the TOS outlined in this document.
+
+### Transfer
+
+You may only transfer the Framework Agreement or Services used in the context of the Framework Agreement to third parties with our prior written consent.
+
+### Term
+
+Coverage under this Agreement will start with Booking of Support Services, for a minimum period of 12 months. Support Services agreements will automatically renew for additional one year terms upon submission of a purchase order for renewal, unless either you or CAOS provides written notice of termination at least 30 days prior to the end of any such term. Each renewal will be at CAOS' then-current rate. In the event that you accesses CAOS Support services in any way after the Agreement has expired or been terminated, you will continue to be bound by this Agreement, which will continue to apply to the services after such expiration or termination.
```
Comment (fforootd) [inline docs/docs/legal/terms-support-service.md]: The mail thing 😄

File: docs/docs/legal/terms-support-service.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+
+This Agreement has the following appendices. When you enter the Agreement with us, you accept these agreements.
+
+* [**Data Processing Agreement**](data-processing-agreement) - How we process personal data on behalf of you
+* [**Support Service Descriptions**](support-services) - How we provide support services to you
+* [**Acceptable Use Policy**](acceptable-use-policy) - What we understand as acceptable and fair use of our Services
+
+### Alterations
+
+Any provisions which deviate from these TOS must be agreed in writing between the Customer and us. Such agreements shall take precedence over the TOS outlined in this document.
```
Comment (fforootd) [inline docs/docs/legal/terms-support-service.md:21]: Mail 😄

File: docs/docs/legal/terms-support-service.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+
+## Your obligations
+
+### Contact information
+
+At our request you will provide your truthful contact information and keep it updated at all times. You must also ensure that you actually receive messages, in particular emails, intended for you.
+
+### Use
+
+You will ensure that the use of our Websites and Services by you or third parties complies with all applicable legislation, these these TOS, and our [Acceptable Use Policy](acceptable-use-policy) at all times.
```
Comment (fforootd) [inline docs/docs/legal/terms-support-service.md:83]:
```suggestion
You will ensure that the use of our Websites and Services by you or third parties complies with all applicable legislation, these TOS, and our [Acceptable Use Policy](acceptable-use-policy) at all times.
```

File: docs/docs/legal/terms-support-service.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+
+Initial know-how transfer for the services will be organized in training sessions conducted by us. We can provide knowledge sessions throughout the term to train newly onboarded staff, update your support staff about important updates, or refresh knowledge in specified areas. In case we notice insufficient quality of support requests from Customers, we will propose appropriate training sessions.
+
+### Security
+
+You will take appropriate measures to prevent any misuse of the services you booked. These include, for example, securing the software used and the prompt installation of security updates as well as using suitably secure passwords.
+
+### Disaster recovery
+
+We take care of the necessary disaster recovery measures. The goal is to maintain a maximum 24h old restore point off all the vital data.
```
Comment (fforootd) [inline docs/docs/legal/terms-support-service.md:105]: we don't provide this with supported units, only dedicated

File: docs/docs/legal/terms-support-service.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+
+### Collection
+
+In the event of default we reserve the right to transfer our claim to a collections agency. You will bear any resulting costs insofar as legally permissible.
+
+### Lapsed Service Fee
+
+In case the term of the Support Service contract has expired within 1 to 180 days, you will be required to pay a Lapsed Service fee in addition to purchasing and activating a one-year renewal contract term at the then-current fee and conditions. The renewal term's start date will also be backdated to begin coverage from the service's original expiration date.
+
+Please contact us for current fees.
```
Comment (fforootd) [inline docs/docs/legal/terms-support-service.md:139]:
```suggestion
Please contact us for the current fees.
```

File: docs/docs/legal/terms-support-service.md [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+
+### Recertification Fee
+
+Recertification of a unit, to be covered under Support Services, is required for:
+
+* units for which Support Services have been expired for more than 180 days
+* units that run a release that is older than 180 days from the products most recent release
```
Comment (fforootd) [inline docs/docs/legal/terms-support-service.md:146]:
```suggestion
* units that run a release that is older than 180 days from the products most recent stable release
```

File: docs/docs/legal/terms-support-service.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+* requests for support on products and services purchased or supported from non-authorized resellers
+
+Recertification of a unit requires payment of a Recertification Fee which results in a [checkup](https://docs.zitadel.ch/docs/guides/installation/checkup) of the unit by CAOS. The unit will be inspected to asses its condition and eligibility for service coverage.
```
Comment (fforootd) [inline docs/docs/legal/terms-support-service.md:149]: Please use a softlink

File: docs/docs/guides/trainings/supportservice/application.md (hunk 2 above)
Comment (mffap) [reply docs/docs/guides/trainings/supportservice/application.md:5]:
```suggestion
## ZITADEL Support
```

File: docs/docs/guides/trainings/supportservice/operations.md (hunk 3 above)
Comment (mffap) [inline docs/docs/guides/trainings/supportservice/operations.md:62]:
```suggestion
```

File: docs/docs/legal/terms-support-service.md [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+
+This Agreement has the following appendices. When you enter the Agreement with us, you accept these agreements.
+
+* [**Data Processing Agreement**](data-processing-agreement) - How we process personal data on behalf of you
+* [**Support Service Descriptions**](support-services) - How we provide support services to you
+* [**Acceptable Use Policy**](acceptable-use-policy) - What we understand as acceptable and fair use of our Services
+
+### Alterations
+
+Any provisions which deviate from these TOS must be agreed in writing between the Customer and us. Such agreements shall take precedence over the TOS outlined in this document.
```
Comment (mffap) [inline docs/docs/legal/terms-support-service.md:21]:
```suggestion
Any provisions which deviate from these TOS must be agreed in writing (mail sufficient) between the Customer and us. Such agreements shall take precedence over the TOS outlined in this document.
```

File: docs/docs/legal/terms-support-service.md (hunk 9 above)
Comment (mffap) [inline docs/docs/legal/terms-support-service.md:105]:
```suggestion
```

File: docs/docs/legal/terms-support-service.md (hunk 12 above)
Comment (mffap) [inline docs/docs/legal/terms-support-service.md:149]:
```suggestion
Recertification of a unit requires payment of a Recertification Fee which results in a [checkup](/docs/guides/installation/checkup) of the unit by CAOS. The unit will be inspected to asses its condition and eligibility for service coverage.
```

File: docs/docs/legal/terms-support-service.md (hunk 10 above)
Comment (mffap) [reply docs/docs/legal/terms-support-service.md:138]: 'the' is not necessary here, as we speak of the general fees. We could say 'contact us for the current lapsed service fee'.

**Review by fforootd — APPROVED** [review]

**Review by fforootd — APPROVED** [review]

**Pull request merged**